ostd/libs/ostd-macros/src/lib.rs

```diff
@@ -44,7 +46,7 @@ pub fn main(_attr: TokenStream, item: TokenStream) -> TokenStream {
 /// For crates other than ostd,
 /// this macro can be used in the following form.
 ///
-/// ```norun
+/// ```compile_fail
```

Comment:
I think `compile_fails` conveys the wrong idea. When a doctest is marked `compile_fails`, you are saying that a wrong way of writing code is being demonstrated here, so it must fail. But our sample code shown here is correct, although the doctest environment cannot run it. So I think the doctest should be marked `ignore`.